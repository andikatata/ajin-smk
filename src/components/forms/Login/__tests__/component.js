import React from 'react';
import renderer from 'react-test-renderer';
import Component from '../component';

jest.mock('redux-form');
jest.mock('../../../../components/elements/TextField');

describe('Component', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<Component />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('renders correctly with message', () => {
    const message = 'test';
    const tree = renderer
      .create(<Component message={message} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});