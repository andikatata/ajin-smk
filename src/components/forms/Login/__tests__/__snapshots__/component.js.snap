// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component renders correctly 1`] = `
<form>
  <div />
  <div />
  <div>
    <span
      className="MuiTypography-root-2 MuiTypography-caption-12 MuiTypography-alignRight-16 Component-text-1"
    >
      Lupa Password?
    </span>
  </div>
  <div>
    <button
      className="MuiButtonBase-root-51 MuiButton-root-27 MuiButton-contained-36 MuiButton-containedPrimary-37 MuiButton-raised-39 MuiButton-raisedPrimary-40 MuiButton-sizeLarge-49 MuiButton-fullWidth-50"
      disabled={false}
      onBlur={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      tabIndex="0"
      type="submit"
    >
      <span
        className="MuiButton-label-28"
      >
        Login
      </span>
      <span
        className="MuiTouchRipple-root-54"
      />
    </button>
  </div>
</form>
`;

exports[`Component renders correctly with message 1`] = `
<form>
  <div />
  <div />
  <div>
    <span
      className="MuiTypography-root-2 MuiTypography-caption-12 MuiTypography-alignRight-16 Component-text-1"
    >
      Lupa Password?
    </span>
  </div>
  <div>
    <button
      className="MuiButtonBase-root-51 MuiButton-root-27 MuiButton-contained-36 MuiButton-containedPrimary-37 MuiButton-raised-39 MuiButton-raisedPrimary-40 MuiButton-sizeLarge-49 MuiButton-fullWidth-50"
      disabled={false}
      onBlur={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      tabIndex="0"
      type="submit"
    >
      <span
        className="MuiButton-label-28"
      >
        Login
      </span>
      <span
        className="MuiTouchRipple-root-54"
      />
    </button>
  </div>
</form>
`;
